raiden-network/light-client
# Feature/dapp mobile optimize bundle size

**Thank you for submitting this pull request :)**

Fixes #2678

**Short description**
Use separate environment files, but also set the Capacitor mode to be productive.

**Definition of Done**


**Steps to manually test the change (dApp)**

1. Run `yarn capacitor:build`
2. Verify that instead of `47MB`, the `./dist` folder is only `17MB` large
